docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -109,6 +109,28 @@ Possible states of the backup:
 - `INCOMPATIBLE`: The backup is incompatible with the current Elasticsearch version.
 - `INCOMPLETE`: The backup is incomplete (this could occur when the backup process was interrupted or individual snapshots were deleted).
 
+## Delete backup API
+
+Note that the backup API can be reached via the `/actuator` management port, which by default is `8092`.  
+An existing backup can be deleted using the below API which deletes all Optimize snapshots associated with the supplied backupID.
+
+```
+DELETE actuator/backup/{backupId}
+```
+
+### Response
+
+| Code             | Description                                                                                                             |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| 204 No Content   | The delete request for the associated snapshots was submitted to Elasticsearch successfully.                            |
+| 500 Server Error | An error occurred, for example the snapshot repository does not exist. Refer to the returned error message for details. |
```

Comment:
What happens if I supply an invalid backupID? Is it also 500?

Reply:
An invalid backupId would always be one where no snapshot exists (since you can't take a backup with an invalid ID), so the delete response would still be submitted but there's just nothing to delete (ie ES will just acknowledge the response and we'll get a 204)